include/mimalloc-internal.h

```diff
@@ -141,9 +141,17 @@ bool        _mi_page_is_valid(mi_page_t* page);
 // Overflow detecting multiply
 #define MI_MUL_NO_OVERFLOW ((size_t)1 << (4*sizeof(size_t)))  // sqrt(SIZE_MAX)
 static inline bool mi_mul_overflow(size_t size, size_t count, size_t* total) {
+#if defined(__GNUC__) || defined(__clang__)
+#if INTPTR_MAX == INT32_MAX
```

Comment:
Thanks!  Can you change this test to: 
```
#if (MI_INTPTR_SIZE==4)
```
to stay consistent with  the rest of the source.